src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -23,7 +24,9 @@ interface Data {
 export class ItemLogViewComponent implements OnChanges {
 
   @Input() itemData?: ItemData;
-  @Input() logColumns?: Column[];
+  @Input() isWatchedGroup = false;
```

Comment:
isWatchingGroup means is user started session for group. Parent component already has it, otherwise I need to do subscription (which I’m not prefer) inside component and change logic a bit.

Reply:
but you still call it `isWatchedGroup` and not `isWatchingGroup`

Reply:
Fixed